boards/shields/ls0xx_generic/doc/index.rst

```diff
@@ -0,0 +1,88 @@
+.. _ls0xx_generic_shield:
```

Comment:
Thank you for the shield support. Can you move shield support to own commit, please?